curvefs/src/tools/query/curvefs_inode_query.cpp

```diff
@@ -0,0 +1,185 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * Project: curve
+ * Created Date: 2022-03-25
+ * Author: chengyi01
+ */
+
+#include "curvefs/src/tools/query/curvefs_inode_query.h"
+
+#include "src/common/string_util.h"
+
+DECLARE_string(metaserverAddr);
+DECLARE_string(poolId);
+DECLARE_string(copysetId);
+DECLARE_string(partitionId);
+DECLARE_string(fsId);
+DECLARE_string(inodeId);
+
+namespace curvefs {
+namespace tools {
+namespace query {
+
+void InodeQueryTool::PrintHelp() {
+    CurvefsToolRpc::PrintHelp();
+    std::cout << " -poolId=" << FLAGS_poolId
+              << " -copysetId=" << FLAGS_copysetId
+              << " -partitionId=" << FLAGS_partitionId
+              << " -fsId=" << FLAGS_fsId << " -inodeId=" << FLAGS_inodeId
+              << " [-metaserverAddr=" << FLAGS_metaserverAddr << "]";
+    std::cout << std::endl;
```

Comment:
I think it's your responsibility to find inode's pool/copyset/partition.

Reply:
> I think it's your responsibility to find inode's pool/copyset/partition.

fix

Reply:
> agree, only fsId and inodeId are needed, and you can get other infos you need inside.

fix